robber/expect.py

```diff
@@ -19,8 +19,9 @@ class expect:
     matchers = {}
     message = None
 
-    def __init__(self, object):
-        self.object = object
+    def __init__(self, obj):
+        self.obj = obj
+        self.is_negative = False
```

Comment:
@catriuspham See this definition: http://www.dictionary.com/browse/negate. `negated` is closer to the meaning of what we are going to do. I think you can find it if you take a look around to another assertion library also.